elixir/apps/web/lib/web/endpoint.ex

```diff
@@ -39,6 +39,7 @@ defmodule Web.Endpoint do
 
   socket "/live", Phoenix.LiveView.Socket,
     websocket: [
+      check_origin: :conn,
```

Comment:
```suggestion
      check_origin: :conn,
```

@andrewdryga this fixed liveview issues in development

Reply:
That's actually a good catch and we do want it on prod, thank you